pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -146,6 +146,22 @@ func (r *ReconcileUserSignup) Reconcile(request reconcile.Request) (reconcile.Re
 	}
 	logger = logger.WithValues("username", userSignup.Spec.Username)
 
+	// ###################################################################################
+	//
+	// VerificationRequired Migration
+	//
+	// TODO remove this after all UserSignup instances have been migrated
+	//
+	// ###################################################################################
+	if userSignup.Spec.VerificationRequired && !states.VerificationRequired(userSignup) {
+		states.SetVerificationRequired(userSignup, true)
+		if err := r.client.Update(context.TODO(), userSignup); err != nil {
+			return reconcile.Result{}, err
+		}
+		// Requeue the reconciliation if the UserSignup was migrated
+		return reconcile.Result{}, nil
```

Comment:
Weill, an empty Result won't trigger a requeue, correct? But that's fine because the spec update will trigger reconcile. So the code is fine but the comment is not quite correct.